### ISSUE TITLE
Need accessibility improvements for screen reader users, can potentially pay for it

### ISSUE THREAD

**Issue by Cleversn** [jitsi/jitsi-meet-electron#164]
Hello,

I work for a company, and we noticed that this Electron version lacks accessibility in some points, e.g. there is no proper labeling on the buttons. Such description labels are important for users like blind people, who use screen readers like NVDA - www.nvaccess.org/.

I offer for doing accessibility testing, and I know a potential funding source, so in case someone can fix the problem, we could pay for that time if it is just a few hours.

**Comment by Echolon** [comment]
@saghul do you have a job board or similar to address this?

**Comment by saghul** [comment]
Closing since this needs to happen in Jitsi Meet and a lot of WCAG work has already happened and more is ongoing.
